thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -565,7 +554,6 @@ class CDistributorBase : public CSimpleInterface, implements IHashDistributor, i
         {
             if (owner.sendBlock(i, msg))
                 return true;
-            atomic_inc(&numFinished);
```

Comment:
I moved it into the caller of CSender::sendBlock (this method), but that does mean that closeWrite() that calls this sendBlock, wouldn't call markStopped()
It wouldn't really matter as it's the end of the process, but it is more correct if markStopped() is called here, so I'm going to change it.